app/controllers/katello/concerns/api/v2/repository_db_content_controller.rb

```diff
@@ -35,15 +35,20 @@ def compare
         respond_for_index(:collection => collection)
       end
 
+      param :available_for, :string, :desc => N_("Show errata that can be added to content view filter")
       def index_relation
         collection = resource_class.scoped
         collection = filter_by_repos(Repository.readable, collection)
         collection = filter_by_repos([@repo], collection) if @repo && !@repo.puppet?
-        collection = filter_by_content_view_filter(@filter, collection) if @filter
         collection = filter_by_content_view_version(@version, collection) if @version
         collection = filter_by_environment(@environment, collection) if @environment
         collection = filter_by_repos(Repository.readable.in_organization(@organization), collection) if @organization
         collection = filter_by_ids(params[:ids], collection) if params[:ids]
+        if params[:available_for] == "content_view_filter"
+          collection = errata_available_for_content_view_filter(@filter, collection) if @filter
+        else
+          collection = filter_by_content_view_filter(@filter, collection) if @filter
+        end
```

Comment:
This approach seems fine, but I'm not sure it belongs in the repository_db_content_controller.  Could we do something like:
```
if params[:available_for] == "content_view_filter" && self.responds_to?(:available_for_content_view_filter)
  colleection = self.available_for_content_view_filter(@filter, collection)
else
  filter_by_content_view_filter(@filter, collection) if @filter
end
```

and then define available_for_content_view_filter within the errata_controller?